packages/api/src/index.ts

```diff
@@ -40,6 +40,9 @@ export type AppState = {
     latest: {
       [empAddress: string]: PriceSample;
     };
+    history: {
+      [empAddress: string]: uma.tables.historicalPrices.SortedJsMap;
```

Comment:
I just realized that I'm unfamiliar with this syntax.

What's the difference between:
```ts
history: {
  [empAddress: string]: uma.tables.historicalPrices.SortedJsMap;
}
```

and 

```ts
history: {
  [key: string]: uma.tables.historicalPrices.SortedJsMap;
}
```

Are `key` and `empAddress` just annotations essentially?

Reply:
yea i think these are just labels, ive called it empAddress for clarity!

Reply:
That's cool! We should do this everywhere!